lib/filclient/filclient.go

```diff
@@ -83,14 +86,37 @@ func (fc *LotusFilClient) VerifyBidder(bidderSig []byte, bidderID peer.ID, stora
 	if err != nil {
 		return false, fmt.Errorf("calling full node state miner info: %v", err)
 	}
-	ownerWalletAddr, err := fc.fullNode.StateAccountKey(ctx, mi.Owner, types.EmptyTSK)
+
+	okOwner, errVerifySigOwner := fc.verifySignature(mi.Owner, sig, bidderID)
+	// Do an early return to avoid double work.
+	if errVerifySigOwner == nil && okOwner {
+		return true, nil
+	}
+
+	okWorker, errVerifySigWorker := fc.verifySignature(mi.Worker, sig, bidderID)
+	if errVerifySigOwner != nil && errVerifySigWorker != nil {
```

Comment:
Don't mean to be picky. The logic seems correct. But for code readability would not be better so that the `if` follows the same pattern as the above `if`?

That would mean 
```
if errVerifySigWorker == nil && okWorker {
	return true, nil
}
```
and the last thing should be the `return false`. The `return true`, `return false`, then `return true` pattern confuses things a bit.

Since this is a critical code hard to test (and you have already tested), I would probably not change it.

Reply:
I think it's totally fine! Might be a bit longer but less "magical" or clearer.

I'll change it!